src/Contexts/quizContext.type.ts

```diff
@@ -0,0 +1,26 @@
+import { Quiz } from "../Database/quiz.type";
+
+export type ContextType = {
+  state: InitialStateType;
+  dispatch: (action: ACTIONTYPE) => void;
```

Comment:
correct me if I'm wrong but isn't the type of dispatch `Dispatch<ACTIONTYPE>`

Reply:
Both ways work Muthu if I am not wrong !